wpilib/wpilib/watchdog.py

```diff
@@ -0,0 +1,83 @@
+# validated: 2018-09-30 EN a818c7fd4741 edu/wpi/first/wpilibj/Watchdog.java
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018 FIRST. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+# ----------------------------------------------------------------------------
+
+import wpilib
+from .timer import Timer
+
+__all__ = ["Watchdog"]
+
+
+class Watchdog:
+    """A class that's a wrapper around a watchdog timer.
+
+    When the timer expires, a message is printed to the console and an optional user-provided
+    callback is invoked.
+
+    The watchdog is initialized disabled, so the user needs to call enable() before use.
+
+    """
```

Comment:
Why is there a blank line before the end quotes here?

Reply:
its what convert_javadoc.html outputs. must have missed that one